rules/windows/command_and_control_remote_file_copy_powershell.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/11/30"
+maturity = "production"
+updated_date = "2020/11/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies powershell.exe being used to download an executable file from a remote destination.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Remote File Download via PowerShell"
+risk_score = 47
+rule_id = "33f306e8-417c-411b-965c-c2812d6d3f4d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "eql"
+
+query = '''
+sequence by host.id, process.entity_id with maxspan=30s
+  [network where process.name : "powershell.exe" and network.protocol == "dns" and
+   not dns.question.name : ("localhost", "*.microsoft.com", "*.azureedge.net", "*.powershellgallery.com", "*.windowsupdate.com", "metadata.google.internal") and 
+   not user.domain : "NT AUTHORITY"]
+  [file where event.type == "creation" and file.extension : ("exe", "dll", "ps1", "bat") and 
```

```suggestion
    [file where process.name : "powershell.exe" and event.type == "creation" and file.extension : ("exe", "dll", "ps1", "bat") and 
```
After some discussion with @brokensound77, we think the addition of the `process.name` value to the file event might tighten up the sequencing to reduce load.